micro_sam/sam_annotator/util.py

```diff
@@ -461,3 +406,157 @@ def _initialize_parser(description, with_segmentation_result=True, with_show_emb
     )
 
     return parser
+
+
+#
+# Helper functions for tracking annotator
```

Comment:
Would it make more sense to put all of the tracking annotator content either in `annotator_tracking.py`, or if it has to be a different file how about a `util_tracking.py` file? This seems like an odd place to put it.

Reply:
Yes, I agree. I also thought about this when refactoring the code. I will see what makes most sense and move this code.